lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -302,6 +302,30 @@ class StandardAccountTest {
         assertTrue(receipt.isAccepted)
     }
 
+    @ParameterizedTest
+    @MethodSource("getAccounts")
+    fun `sign and send declare v2 transaction (cairo compiler v2)`(accountAndProvider: AccountAndProvider) {
+        val (account, provider) = accountAndProvider
+        val contractCode = Path.of("src/test/resources/compiled_v2/${provider::class.simpleName}_new_syntax_test_contract.json").readText()
```

Comment:
```suggestion
        val contractCode = Path.of("src/test/resources/compiled_v2/${provider::class.simpleName}_contract.json").readText()
```

new sytnax is something that may soon be not true ( ͡° ͜ʖ ͡°)

Reply:
the same change for casmCode below

Reply:
That's the name in cairo repo. But the fact that the name soon won't be true is true as well.